tools/mpremote/mpremote/main.py

```diff
@@ -324,10 +335,7 @@ def argparse_none(description):
         ],
         "help": "make the device enter its bootloader",
     },
-    "setrtc": [
-        "exec",
-        "import machine; machine.RTC().datetime((2020, 1, 1, 0, 10, 0, 0, 0))",
-    ],
+    "setrtc": "rtc --set",  # Backwards compatible alias for old command.
```

Comment:
Since this never really worked as intended (it set a fixed time...) I think it's safe to just delete it, and the user can add their own shortcut if needed (and this new `setrtc` alias is not doing the same thing as the old command).

Reply:
OK!